lib/tasks/evm_dba.rake

```diff
@@ -195,9 +179,10 @@ namespace :evm do
 
         puts "Initializing region and database..."
         # Create the region from our REGION file, initialize a new miq_database row for this region
-        EvmDba.run_cmd_with_open4("bin/rails runner 'MiqDatabase.seed; MiqRegion.seed;'")
+        AwesomeSpawn.run!("bin/rails runner", :params => ["MiqDatabase.seed; MiqRegion.seed"])
       rescue => err
-        STDERR.puts "Encountered issue setting up Database using region #{region}: #{err}\n"
+        message = err.kind_of?(AwesomeSpawn::CommandResultError) ? err.error : err.message
```

Comment:
> That sounds like a UI level concern to me

Well, the "UI" in this case is our logs.  So stuff runs, goes into the logs, and then we get logs from users for diagnostics.  For things like a database, a user can scrub it of decryptable passwords.  There's no such mechanism for random text in a log.